server/src/main/java/org/candlepin/controller/PoolManager.java

```diff
@@ -190,7 +190,8 @@ Entitlement adjustEntitlementQuantity(Consumer consumer, Entitlement entitlement
      */
     Page<List<Pool>> listAvailableEntitlementPools(Consumer consumer, ActivationKey key,
         Owner owner, String productId, String subscriptionId, Date activeOn, boolean activeOnly,
-        boolean includeWarnings, PoolFilterBuilder filterBuilder, PageRequest pageRequest);
+        boolean includeWarnings, PoolFilterBuilder filterBuilder, PageRequest pageRequest,
```

Comment:
if activeOnly is never set to False, then this parameter should be stripped out from the entire stack.